lib/rucio/web/rest/flaskapi/v1/common.py

```diff
@@ -210,6 +210,15 @@ def parse_scope_name(scope_name: str, vo: Optional[str]) -> tuple[str, ...]:
     :raises ValueError: when scope_name could not be parsed.
     :returns: a (scope, name) tuple.
     """
+
+    encoded_slash = r"%2F"
+    if encoded_slash in scope_name:
```

Comment:
Since we're server-side, I wonder if it would be faster to check if `AllowEncodedSlashes NoDecode` is true

Reply:
It is a possibility, but I prefer this appraoch which is not dependent on the apache server, but it's a matter of preference. How would you check this in a reliable manner?

Reply:
Updated, let me know if you agree with this way of checking for the setting.